### PR TITLE
da: copy mostly trivial code blocks

### PR DESCRIPTION
**Pull request by mgeisler**
Most of these code blocks don't have strings or comments to translate (but a few do).

Part of #286.

### CODE REVIEW

**Comment by mgeisler** [comment]
Hejsa, kan jeg få et review her? :slightly_smiling_face:

File: po/da.po [hunk 1]
```diff
@@ -3063,6 +3187,29 @@ msgid ""
...
+"    }\n"
+"\n"
+"    fn inc_width(&mut self, delta: u32) {\n"
+"        self.width += delta;\n"
+"    }\n"
+"}\n"
+"\n"
+"fn main() {\n"
+"    let mut rect = Rectangle { width: 10, height: 5 };\n"
+"    println!(\"gammels areal: {}\", rect.area());\n"
```
Comment (thedataking) [inline po/da.po:3208]: gammelt?

**Review by thedataking — APPROVED** [review]
LGTM other than one little papercut

File: po/da.po (hunk 1 above)
Comment (mgeisler) [reply po/da.po:3208]: Doh! :smile: 

```suggestion
"    println!(\"gammelt areal: {}\", rect.area());\n"
```

**Comment by mgeisler** [comment]
> LGTM other than one little papercut

Thanks!

**Pull request merged**